migrations/2025-06-04-103308_multi-community/up.sql

```diff
@@ -75,11 +75,17 @@ WHERE
 ALTER TYPE listing_type_enum
     ADD VALUE 'Suggested';
 
-CREATE INDEX idx_multi_community_read_from_name ON multi_community (local, deleted, name);
+CREATE INDEX idx_multi_community_read_from_name ON multi_community (local)
+WHERE
+    local AND NOT deleted;
 
 CREATE INDEX idx_multi_community_ap_id ON multi_community (ap_id);
 
 CREATE INDEX idx_multi_creator_id ON multi_community (creator_id);
 
+CREATE INDEX idx_multi_community_follow_multi_id ON multi_community_follow (multi_community_id);
+
 CREATE INDEX idx_multi_followed_by ON multi_community_follow (person_id);
 
```

Comment:
```suggestion
```

This index on `(person_id)` is only a waste of resources, so it should be removed.

The new primary key automatically creates an index on `(person_id, multi_community_id)`, which completely removes the need for the `(person_id)` index. Both indexes have `person_id` all the way to the left side of the column list, so a query with a filter on `person_id` can use either index to avoid scanning rows that don't have a matching `person_id` value.